### DISCUSSION TITLE
Publishing this app as a standalone offline executable with electron js

### DISCUSSION THREAD

**Discussion by MohammadMD1383** [excalidraw/excalidraw#4565] [Show and tell]
After running `npm run build` I used `electron build/index.html` to start the application.
And that was fully working!
I think that would be a very nice thing for students or teachers who want to use this whiteboard offline.

**Comment by dwelle** [comment]
We actually used to publish an electron app, but deprecated it in favor of a PWA. Read more on https://blog.excalidraw.com/deprecating-excalidraw-electron/

**Reply by MohammadMD1383** [reply]
well, you're right
